variables.tf

```diff
@@ -35,11 +35,16 @@ variable "subnets" {
   description = "List of VPC subnet IDs"
 }
 
-variable "availability_zones" {
+variable "cluster_availability_zones" {
   type        = "list"
   description = "List of Availability Zones that instances in the DB cluster can be created in"
 }
 
+variable "instance_availability_zones" {
```

Comment:
1. instance AZ is single, not a list
https://www.terraform.io/docs/providers/aws/r/rds_cluster_instance.html#availability_zone

2. It's computed by AWS, not sure if we need to specify it

Reply:
This was required if you don't want to distribute the autoscaling nodes within the azs. If you use default computed value, terraform will always recreate the rds instances with showing changes in azs.